payments-core/src/test/java/com/stripe/android/payments/financialconnections/GetFinancialConnectionsAvailabilityTest.kt

```diff
@@ -1,19 +1,32 @@
 package com.stripe.android.payments.financialconnections
 
 import com.stripe.android.core.utils.FeatureFlags.financialConnectionsFullSdkUnavailable
-import com.stripe.android.core.utils.FeatureFlags.financialConnectionsLiteKillswitch
+import com.stripe.android.core.utils.FeatureFlags.financialConnectionsLiteEnabled
 import com.stripe.android.model.ElementsSession
 import com.stripe.android.model.ElementsSession.Flag.ELEMENTS_DISABLE_FC_LITE
+import com.stripe.android.testing.FeatureFlagTestRule
+import org.junit.Rule
 import org.mockito.kotlin.doReturn
 import org.mockito.kotlin.mock
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
 class GetFinancialConnectionsAvailabilityTest {
 
+    @get:Rule
+    val financialConnectionsFullSdkUnavailableFeatureFlagTestRule = FeatureFlagTestRule(
+        featureFlag = financialConnectionsFullSdkUnavailable,
+        isEnabled = false
+    )
+
+    @get:Rule
+    val financialConnectionsLiteEnabledFeatureFlagTestRule = FeatureFlagTestRule(
+        featureFlag = financialConnectionsLiteEnabled,
+        isEnabled = false
+    )
```

Comment:
You only need one rule, and can then call `setEnabled()` on the rule in the test where the value is different from the default.

Reply:
but you need one rule per flag right?